test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -11079,25 +11090,45 @@ def add_server_dyn_res(self, custom_resource, script_body=None,
         :param validate: if True (the default), validate the
                          configuration settings.
         :type validate: bool
+        :param dirname: the file will be created in this directory
+        :type dirname: str or None
+        :param host: the hostname on which dyn res script is created
+        :type host: str or None
+        :param perm: perm to use while creating scripts
+                     (must be octal like 0777)
+        :param prefix: the file name will begin with this prefix
+        :type prefix: str
+        :param suffix: the file name will end with this suffix
+        :type suffix: str
```

Comment:
Please add a :return to describe what this function returns since it seems to return the name of the file created.